src/html_files/index.html

```diff
@@ -24,6 +24,7 @@
 			<button class="tablinks" name="interrupts">Interrupt Data</button>
 			<button class="tablinks" name="disk_stats">Disk Stats</button>
 			<button class="tablinks" name="netstat">Net Stats</button>
+			<button class="tablinks" name="aperfrunlog">Aperf Runlog</button>
```

Comment:
We're already at too many tabs. They can be vertically shrunk some to buy time, but I think we'll have to start collapsing some of this. For example, just have a single 'Internals' tab, under which there are the aperf internal stats and now runlog (possibly via a sub-tab).